### PR TITLE
switches from not CA signed to self CA signed for streaming TLS

### PR DESCRIPTION
**Pull request by mikebrow**
Addressing issue: #712

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

**Comment by Random-Liu** [comment]
@mikebrow Please rebase this.

File: pkg/server/streaming.go [hunk 1]
```diff
@@ -139,54 +133,27 @@ func handleResizing(resize <-chan remotecommand.TerminalSize, resizeFunc func(si
...
-	// Generate a new random serial number for certificate
-	serialNumber, err := rand.Int(rand.Reader, new(big.Int).Lsh(big.NewInt(1), 128))
-	if err != nil {
-		return fail(errors.Wrap(err, "failed to generate serial number"))
-	}
 	hostName, err := os.Hostname()
 	if err != nil {
 		return fail(errors.Wrap(err, "failed to get hostname"))
 	}
+	host := fmt.Sprintf("%s:%s:%s", certOrganizationName, certCommonName, hostName)
```
Comment (Random-Liu) [inline pkg/server/streaming.go]: It seems that we don't to specify org and common name, the util function has done that for us.

Comment (mikebrow) [reply pkg/server/streaming.go]: done...

**Comment by mikebrow** [comment]
@Random-Liu rebased..

**Pull request merged**